src/main/java/org/opentripplanner/graph_builder/module/ned/ElevationModule.java

```diff
@@ -547,15 +639,47 @@ private double getElevation(double x, double y) throws PointOutsideCoverageExcep
             // for DefaultGeographicCRS.WGS84, but OTP is using (long, lat) throughout and assumes unprojected DEM
             // rasters to also use (long, lat).
             DirectPosition2D projectedPosition = new DirectPosition2D(GeometryUtils.WGS84_XY, x, y);
-            DirectPosition2D transformedPosition = new DirectPosition2D();
-            transformer.transform(projectedPosition, transformedPosition);
-            coverage.evaluate(transformedPosition, values);
+            DirectPosition2D position2D;
+            // If this is not done during testing, a downstream library will raise an assertion error
+            if (transformCoordinates) {
```

Comment:
Again, I think this may be unnecessary.

Reply:
I swear I was seeing assertion errors when I started testing this, but now I'm no longer seeing them. I'll remove this now.